Topten.RichTextKit/TextShaping/TextShaper.cs

```diff
@@ -15,7 +15,6 @@
 
 using HarfBuzzSharp;
 using SkiaSharp;
-using SkiaSharp.HarfBuzz;
```

Comment:
This really is just an extra dependency that is not needed for this type of library. Just a few lines and we are up and running.